MainCore/Tasks/Misc/SendResourcesInTask.cs

```diff
@@ -0,0 +1,457 @@
+                var result = ClickSendResources();
+                if (result.IsFailed) return result.WithError(new Trace(Trace.TraceMessage()));
+            }
+
+            {
+                var result = ClickSend();
+                if (result.IsFailed) return result.WithError(new Trace(Trace.TraceMessage()));
+            }
+            // TODO: Could schedule village refresh when resources arrive. Optional.
+
+            return Result.Ok();
+        }
+
+        private Result Update()
+        {
+            using var context = _contextFactory.CreateDbContext();
+
+            var decider = DateTime.Now.Ticks % 2 == 0;
+
+            if (decider)
+            {
+                _navigateHelper.ToDorf2(AccountId);
+                _navigateHelper.SwitchVillage(AccountId, VillageId);
+            }
+            else
+            {
+                _navigateHelper.SwitchVillage(AccountId, VillageId);
+                _navigateHelper.ToDorf2(AccountId);
+            }
+            {
+                var result = _updateHelper.UpdateDorf2(AccountId, VillageId);
+                if (result.IsFailed) return result.WithError(new Trace(Trace.TraceMessage()));
+            }
+
+            return Result.Ok();
+        }
+
+        private Result ToMarketPlace()
+        {
+            using var context = _contextFactory.CreateDbContext();
+            var marketplace = context.VillagesBuildings.Where(x => x.VillageId == this._sendFromVillageId).FirstOrDefault(x => x.Type == BuildingEnums.Marketplace && x.Level > 0);
+            if (marketplace is null)
+            {
+                _logManager.Information(AccountId, "Marketplace is missing. Turn off auto Sending Resources to prevent bot detector.");
+                var setting = context.VillagesMarket.Find(VillageId);
+                setting.IsGetMissingResources = false;
+                context.Update(setting);
+                context.SaveChanges();
+                return Result.Fail(new Skip());
+            }
+            {
+                var result = _navigateHelper.GoToBuilding(AccountId, marketplace.Id);
+                if (result.IsFailed) return result.WithError(new Trace(Trace.TraceMessage()));
+            }
+            {
+                var result = _navigateHelper.SwitchTab(AccountId, 1);
+                if (result.IsFailed) return result.WithError(new Trace(Trace.TraceMessage()));
+            }
+
+            return Result.Ok();
+        }
+
+        private Result EnterCoordinates()
+        {
+            var html = _chromeBrowser.GetHtml();
+
+            var xCoordinate = html.GetElementbyId("xCoordInput");
+            var yCoordinate = html.GetElementbyId("yCoordInput");
+
+            if (xCoordinate is null)
+            {
+                throw new Exception("Destination village coordinates not found.");
+            }
+            if (yCoordinate is null)
+            {
+                throw new Exception("Destination village coordinates not found.");
```

Comment:
make a _logManager.Information and return Result.Fail(new Skip()) instead throwing exception

Reply:
there is some place like this change them too